packs/bit-bucket/actions/list_user_repo.py

```diff
@@ -0,0 +1,21 @@
+from lib.action import BitBucketAction
+
+
+class GetRepos(BitBucketAction):
+    def run(self):
+        """
+        Listing repositories for a user.It assumes
+        that you have already places the name and
+        password in config file. It returns repo's
+        name, its state and private status(returns
+        True if its private, False otherwise)
+        """
+        repository = []
+        bb = self.perform_request()
+        success, repos = bb.repository.all()
+        for repo in sorted(repos):
+            repository.append(
+                {repo['name']: {'state': repo['state'],
```

Comment:
You should decide to either return a list of repo objects or an object (dict) where the key is a repo name and the value are other repo attributes.

It makes little sense to have another level of indirection and return a list of nested objects (nesting under repo name in this case doesn't provide any value).